ocaml/xapi/xapi_pool.ml

```diff
@@ -1892,6 +1892,17 @@ let eject_self ~__context ~host =
       | `Static ->
           "static"
     in
+    let mode_v6 =
+      match pif.API.pIF_ipv6_configuration_mode with
+      | `None ->
+          "none"
+      | `DHCP ->
+          "dhcp"
+      | `Static ->
+          "static"
+      | `Autoconf ->
+          "autoconf"
```

Comment:
Let's avoid ad-hoc conversion methods, and use autogenerated ones, which are always up-to-date:
```suggestion
      Record_util.ipv6_configuration_mode_to_string
        pif.API.pIF_ipv6_configuration_mode
      |> String.uncapitalize_ascii
```